documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -30,23 +31,23 @@ export default {
 };
 ```
 
-...and add the [`prerender`](page-options#prerender) option to your root layout:
+...et ajoutez l'option [`prerender`](page-options#prerender) à votre <span class="vo">[layout](PUBLIC_SVELTE_SITE_URL/docs/web#layout)</span> racine :
 
 ```js
 /// file: src/routes/+layout.js
-// This can be false if you're using a fallback (i.e. SPA mode)
+// Cette valeur peut être `false` si vous utilisez le mode SPA
 export const prerender = true;
 ```
 
-> You must ensure SvelteKit's [`trailingSlash`](page-options#trailingslash) option is set appropriately for your environment. If your host does not render `/a.html` upon receiving a request for `/a` then you will need to set `trailingSlash: 'always'` in your root layout to create `/a/index.html` instead.
+> Vous devez vous assurer que l'option de SvelteKit [`trailingSlash`](page-options#trailingslash) est correctement définie pour votre environnement. Si votre hôte ne renvoie pas `/a.html` pour requête demandant `/a`, vous aurez alors besoin de définir `trailingSlash: 'always'` dans votre <span class="vo">[layout](PUBLIC_SVELTE_SITE_URL/docs/web#layout)</span> racine pour renvoyer à la place `/a/index.html`.
 
-## Zero-config support
+## Support sans configuration
 
-Some platforms have zero-config support (more to come in future):
+Certaines plateformes n'ont pas besoin de configuration (d'autres sont à venir) :
```

Comment:
```suggestion
Certaines plateformes n'ont pas besoin de configuration (plus d'informations à ce sujet à venir) :
```

je ne suis pas sur que "more to come in future" soit utilisé dans ce sens là. C'est pas plutôt "plus d'infos dans le future" ?

Reply:
hm, non je pense vraiment qu'il parle du fait que d'autres plateformes que Vercel pourront bientôt être utilisées sans configuration. Le fait qu'il n'y ait qu'une seule entrée dans cette liste va dans ce sens là.